www/js/goals.js

```diff
@@ -891,20 +973,22 @@ angular.module('emission.main.goals',['emission.services', 'emission.plugin.logg
       nzTour.start(tour);
     };
 
+    $scope.startWalkthrough = function () {
+      startWalkthrough();
+    }
+
     /*
     * Checks if it is the first time the user has loaded the goals tab. If it is then
     * show a walkthrough and store the info that the user has seen the tutorial.
     */
-    var checkGoalsTutorialDone = function () {
+    /*var checkGoalsTutorialDone = function () {
       var GOALS_DONE_KEY = 'goals_tutorial_done';
       var goalsTutorialDone = storage.get(GOALS_DONE_KEY);
       if (!goalsTutorialDone) {
         startWalkthrough();
         storage.set(GOALS_DONE_KEY, true);
       }
-    };
+    };*/
 
-    $scope.startWalkthrough = function () {
-      startWalkthrough();
-    }
+    //checkSurveyDone();
```

Comment:
I had to comment out the call to checkSurveyDone() because it makes the app crash when I go to the Goals tab. Let me know if you can see what I'm doing wrong...